### PR TITLE
feat(content): New uninhabited planet: Donya

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Content (Planet)**

## Summary
Added a new planet in Zubenelgenubi called Donya. It has poisonous rain and uses one of the new landscape images added in #10261.

This would be one of the colonies that the "billionaire philanthropists" tried to start. However, here they were not as lucky as on planets like Zug.

## Screenshots
donya

The landscape image and description of the planet.

map

Where it is on the map.

## Testing Done
Took screenshot, landed, launched, landed, etc.

## Save File
This save file can be used to test these changes:
[Asiah Racer~Donya.txt](https://github.com/user-attachments/files/17199144/Asiah.Racer.Donya.txt)

Load the save, take off, jump, and land. The travel course is already mapped out.

### CODE REVIEW

File: data/map planets.txt [hunk 1]
```diff
@@ -1099,6 +1099,12 @@ planet "Dlia Jzaur"
 	description `	Oddly enough, Dlia Jzaur has many unnatural looking formations on the surface that seem to reach up to the sky in areas where eclipses often occur. The formations are hollow underneath, but show no signs of Gegno involvement.`
 	government Uninhabited
 
+planet "Donya"
+	attributes uninhabited
+	landscape land/fields29
+	description `Donya is a lush green world much like Earth, filled with vegetation and growing matter. It even has clouds and a blue sky, making it idyllic to the eyes of settlers, who mostly had been experiencing unearthly conditions on other planets. However, it remains unsettled, because the clouds release sheets of poisonous acid on a regular timeframe. Between these moments of unrest the planet is completely peaceful. The trees and grasses planted by the initial wave of colonists seem to have adjusted to the terrible rain quite well, unlike their masters, who left at the first opportunity.`
```
Comment (Koranir) [inline data/map planets.txt:1105]:
```suggestion
	description `Donya is a lush, green, Earth-like world filled with vegetation and growing matter. With clouds and a blue sky, it was idyllic to the eyes of early settlers. Despite that, it remains unsettled as rains of poisonous acid fall on regular timeframes, rendering the planet uninhabitable to human life. The foliage planted by the initial wave of colonists seems to have adjusted to the terrible rain quite well unlike their masters, who left at the first opportunity.`
```

**Review by Koranir — CHANGES_REQUESTED** [review]
I'm not quite sure why this planet needs to exist, to be honest.

**Comment by TheGiraffe3** [comment]
I thought we were trying to create more uninhabited planets (like #9917 did).

**Comment by Koranir** [comment]
> If that's not the case, I can close this.

No need, I just wanted to know the motive.

File: data/map planets.txt [hunk 2]
```diff
@@ -1099,6 +1099,12 @@ planet "Dlia Jzaur"
 	description `	Oddly enough, Dlia Jzaur has many unnatural looking formations on the surface that seem to reach up to the sky in areas where eclipses often occur. The formations are hollow underneath, but show no signs of Gegno involvement.`
 	government Uninhabited
 
+planet "Donya"
+	attributes uninhabited
+	landscape land/fields29
+	description `Donya is a lush, green, Earth-like world filled with vegetation and growing matter. With clouds and a blue sky, it was idyllic to the eyes of early settlers. Despite that, it remains unsettled as rains of poisonous acid fall on regular timeframes, rendering the planet uninhabitable to human life. The foliage planted by the initial wave of colonists seems to have adjusted to the terrible rain quite well unlike their masters, who left at the first opportunity.`
```
Comment (mOctave) [inline data/map planets.txt:1105]: Okay, so it's filled with growing matter but simultaneously has a bad case of acid rain? That doesn't make any sense. Generally, any acid that is dangerous to humans is also dangerous to plants and animals. Is there some sort of radiation or something causing the foliage to mutate, creating a monoculture, and also increasing the risk to humans? Was it super genetically engineered before it was planted, and supposed to scrub the acid, but failed to do so?

File: data/map planets.txt [hunk 3]
```diff
@@ -1099,6 +1099,12 @@ planet "Dlia Jzaur"
 	description `	Oddly enough, Dlia Jzaur has many unnatural looking formations on the surface that seem to reach up to the sky in areas where eclipses often occur. The formations are hollow underneath, but show no signs of Gegno involvement.`
 	government Uninhabited
 
+planet "Donya"
+	attributes uninhabited
+	landscape land/fields29
```
Comment (mOctave) [inline data/map planets.txt:1104]: Is this the right image for this planet? The short grass implies human habitation.

**Review by mOctave — COMMENTED** [review]
Grammar's pretty good, and file formatting's great, but I'm a bit confused about the whole acid rain thing...

File: data/map planets.txt (hunk 3 above)
Comment (TheGiraffe3) [reply data/map planets.txt:1104]: I was thinking (and I guess this wasn't reflected in the description) that the soil has nutrients that helped against the rain. The grasses get melted by the rain down until that specific point.

File: data/map planets.txt (hunk 2 above)
Comment (TheGiraffe3) [reply data/map planets.txt:1105]: See below. Also I figured that the trees were better at getting nutrients from the soil...
But probably genetically engineered.

Comment (TheGiraffe3) [reply data/map planets.txt:1105]: Does this look better?
Might be too long though.
```suggestion
	description `Donya is a lush, green, Earth-like world filled with vegetation and growing matter. With clouds and a blue sky, it was idyllic to the eyes of early settlers who were being funded by billionaire philanthropists.`
	description `	Despite the beauty and the funding, it remains unsettled, as rains of poisonous acid fall on regular timeframes, rendering the planet uninhabitable to human life. The foliage planted by the initial wave of colonists to scrub the skies of the rain seems to have adjusted to the terrible liquid quite well, although it hasn't managed to entirely eliminate it. The planters, who couldn't bear the dreadful weather, left when it was discovered that their plan would not work.`
```

File: data/map planets.txt (hunk 3 above)
Comment (mOctave) [reply data/map planets.txt:1104]: Interesting... so the original colonists created the perfect landscaping system, but then couldn't control it? I like that idea a lot.

File: data/map planets.txt (hunk 2 above)
Comment (mOctave) [reply data/map planets.txt:1105]: Pretty good, although a bit wordy in places.
```suggestion
	description `Donya is a lush, green, Earth-like world filled with vegetation and growing matter. With clouds and a blue sky, it was idyllic in the eyes of early settlers and the billionaire philanthropists who funded them.`
	description `	Despite the beauty and the funding, it remains unsettled, as rains of poisonous acid fall on regular timeframes, rendering the planet uninhabitable to human life. The foliage planted by the initial wave of colonists to scrub the skies of the rain seems to have adjusted to the terrible liquid quite well, although it hasn't managed to entirely eliminate it. The planters, who couldn't bear the dreadful weather, left when they discovered that they couldn't stop the rains.`
```

**Comment by Anarchist2** [comment]
#9917 (and #9879) were created primarily to fix #4628, by adding more places for pirates to land and save their game. Donya doesn't contribute much on this end; it's sandwiched between Rowark's Roost, Hope and Bloodsea.

Right now, I'd rather that we build upon our existing planets (both inhabited and uninhabited) rather than adding more uninhabited planets that don't really do anything.

**Comment by TheGiraffe3** [comment]
> #9917 (and #9879) were created primarily to fix #4628, by adding more places for pirates to land and save their game. Donya doesn't contribute much on this end; it's sandwiched between Rowark's Roost, Hope and Bloodsea.

But for players who are wanting to go toward Gamma Corvi for some reason or something like that... it would also be possible to move the planet. I just don't see a problem with adding another world.

**Comment by Saugia** [comment]
>But for players who are wanting to go toward Gamma Corvi for some reason or something like that... it would also be possible to move the planet. I just don't see a problem with adding another world.

Well, adding a planet for the sake of adding a planet isn't a benefit to the game in itself. Like some other comments above, there should be a good reason to add a new planet outside of "more uninhabited planets good". So, with Anarachist's comment above, this isn't really a good place for a new planet, but there might be another place it can go, yes.

What that is, I can't think of at the moment, but it's not that "just adding another world" is a problem, it's adding one with no reason that is.

**Comment by TheGiraffe3** [comment]
I'll close this for now then.
